packages/amplify-schema-validator/README.md

```diff
@@ -0,0 +1,29 @@
+# Amplify GraphQL Schema Validator
+
+This package contains logic used by AWS Amplify development teams to validate
+that customer's `schema.graphql` for their Amplify app is valid.
+
+## Contributing
+
+To add a validator, simply create a new `*.ts` file under `src/validators`
+exporting a function named `validate` that takes a `DocumentNode` object and
+returns true if successful, and throws an actionable exception if not.
+
+Validators should be pure functions that do not depend on or mutate state.
+
+Once your validator is ready, update `src/index.ts` and import your validator
+there.
+
+Ensure your validator is tested and conforms to our coverage standards.
+
+### Installing dependencies
+
+`npm install`
+
+### Running tests
+
+`npm run test`
+
+### Building the package
+
+`npm run build`
```

Comment:
As long as this package lives under the `packages` directory I wouldn't recommend this method since this lives in a monorepo, and we don't want multiple lockfiles. yarn, yarn test, and yarn build from root should cover these.

Reply:
I'll remove this section since it's covered by the root level readme / contributing guide